src/OrchardCore/OrchardCore.DisplayManagement.Liquid/LiquidViewsFeatureProvider.cs

```diff
@@ -59,7 +64,10 @@ public void PopulateFeature(IEnumerable<ApplicationPart> parts, ViewsFeature fea
                     feature.ViewDescriptors.Add(new CompiledViewDescriptor
                     {
                         RelativePath = viewPath,
-                        Item = new RazorViewCompiledItem(typeof(LiquidPage), @"mvc.1.0.view", viewPath)
+                        Item = new RazorViewCompiledItem(
+                            typeof(LiquidPage),
+                            MvcViewDocumentClassifierPass.MvcViewDocumentKind,
```

Comment:
To make it even clearer because it's not obvious what I am asking here (I could do it but I prefer this is well understand) here is what the "generated" code is for a razor view: https://github.com/dotnet/aspnetcore/blob/main/src/Mvc/Mvc.Analyzers/test/TagHelpersInCodeBlocksAnalyzerTest.cs#L673

The idea is that now each generated view contains its own string instance when instantiated. What needs to happen is that the "generated source code" needs to use a static reference. That's why I mean to pass the string containing the name of the static reference. And I am assuming here that the argument is the one that will be used for the code generation. I don't remember how we can check what is actually generated (maybe ibn /obj files)

Reply:
Okay I got the idea, interesting I will look at it this night